recipe/conda_build_config.yaml

```diff
@@ -5,23 +5,23 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
 folly_build_ext:
   - None
   - "jemalloc"  # [not win]
-fmt:
-  - "10"
-  - "9"
-  - "9"
+libboost_devel:
+  - "*"
```

Comment:
What's the reason for not pinning here?

Reply:
Indeed, we need the entire cross-product.